src/tfont/objects/path.py

```diff
@@ -257,6 +257,8 @@ def __repr__(self):
     def bounds(self):
         points = self.points
         pts_len = len(points)
```

Comment:
Here you can do `type = self._points[self._end].type` instead of using len.

Reply:
Why not `self.type`?